Makefile

```diff
@@ -34,6 +34,9 @@ SCALE_FACTOR ?= 2
 endif
 TIMEOUT_RETRIES ?= $$((${TIMEOUT_M} * ${SCALE_FACTOR} * (${RETRY} + 1) ))m
 CRE ?= podman
+# avoid localized error messages (that are matched against in certain cases)
+LANG = C.utf8
+LANGUAGE =
```

Comment:
According to https://www.gnu.org/software/gettext/manual/html_node/Locale-Environment-Variables.html#Locale-Environment-Variables one should be able to use `LC_ALL` instead of needing to specify both LANG+LANGUAGE

Reply:
This doesn't work because `LANGUAGE` is still used when it is set (unless `LANG` or `LC_ALL` is set to `C`). I've tested that so `LANGUAGE` definitely needs to be unset manually. However, I now changed it to `LC_ALL` instead of `LANG` because it has indeed a higher precedence than `LANG` and is also used by autotools.